src/ui/src/pages/app/bootstrap_monitors/Index/components/MonitorForm/index.tsx

```diff
@@ -80,6 +82,11 @@ const MonitorForm: React.FC = () => {
           <Form.Control type="text" placeholder="TPCH_SF1000" disabled value={schema} />
         </Form.Group>
 
+        <Form.Group className="mb-3" controlId="formBasicSchema">
+          <Form.Label>Lookback Days (# Days Ago to Fetch)</Form.Label>
+          <Form.Control type="number" value={daysAgo} onChange={(e: any) => setDaysAgo(e.target.value)} />
```

Comment:
We should be able to type this as `HTMLInputElement` just so we avoid `any`

Reply:
Same as above